lib/bolt/config.rb

```diff
@@ -484,6 +484,10 @@ def validate
         Bolt::Util.validate_file('inventory file', default_inventoryfile)
       end
 
+      if @data['plugin-cache'] && @data['plugin-cache']['ttl'] < 0
+        raise Bolt::ValidationError, "plugin-cache.ttl must be a positive integer"
+      end
```

Comment:
```suggestion
      if @data['plugin-cache'] && @data['plugin-cache']['ttl'] < 0
        raise Bolt::ValidationError, "plugin-cache.ttl must be a positive integer"
      end
```

This can be handled by setting `:minimum` in the schema.